src/mocks/event.js

```diff
@@ -0,0 +1,20 @@
+import Moment from 'moment';
+
+export default {
+    title: 'Steve\'s BBQ',
+    location: {
+        lat: 49.2827,
+        lon: -123.1207
+    },
+    description: `
+    I got 10,000 steaks in an Air Miles scam gone wrong so 
```

Comment:
Or, an Air Miles scam gone _extremely_ right

Reply:
Also, do we think HTML or text for the description is preferable? We get both from Eventbrite so it's all the same on the backend